### PR TITLE
Removes Unused Species

### PR DESCRIPTION
**Pull request by DatBoiTim**
## About The Pull Request

I'm working towards species singletons. Removing some bloat species would vastly decrease the amount of stuff I need to do.
Removes Unused(beyond admin intervention) Species that are also poorly maintained or completely pointless.

These species are attainable via wabbajack(and are unlikely given the number of species) but the availability of the wabbajack is so limited, these only saw the light of day under admin use(which may as well be never).

Affected Species:
Corporate Agent(Basically just a say_mod as a species, unused and unnecessary)
Mushroompeople(Unmaintained, unused) the martial art still remains if shenanigans are desired.

## Why It's Good For The Game
Bloat is bad for long term development. Underutilized code.

Complete code removal

## Changelog
:cl: DatBoiTim
del: Removed Corporate Agents
del: Removed Mushroom Species
/:cl:

### CODE REVIEW

**Comment by DatBoiTim** [comment]
Crossed Talks About Bloat Reduction

**Comment by DatBoiTim** [comment]
Ike has also voiced approval

**Comment by Sinestia** [comment]
This doesn't even remove the most popular wiz species so I can't see how anyone could be angery about this

**Comment by DatBoiTim** [comment]
> This doesn't even remove the most popular wiz species so I can't see how anyone could be angery about this

Being?

File: code/modules/awaymissions/mission_code/snowdin.dm [hunk 1]
```diff
@@ -207,7 +207,7 @@
 					var/list/robo_parts = list()//keep a reference of robotic parts so we know if we can turn them into a plasmaman
 					var/mob/living/carbon/human/PP = L
 					var/S = PP.dna.species
-					if(istype(S, /datum/species/plasmaman) || istype(S, /datum/species/android)) //ignore plasmamen/robotic species
+					if(istype(S, /datum/species/plasmaman)) //ignore plasmamen/robotic species
```
Comment (ike709) [inline code/modules/awaymissions/mission_code/snowdin.dm]:
```suggestion
					if(istype(S, /datum/species/plasmaman)) //ignore plasmamen
```

**Review by ike709 — DISMISSED** [review]

**Comment by dannydonk** [comment]
Android species is used by chaplain's mechanicus sect.

**Comment by DatBoiTim** [comment]
> Android species is used by chaplain's mechanicus sect.

"Androids(Unmaintained, barely used)"

These are far less common than the fireproofing ritual, and even on rounds where the sect is seen, there's maybe a few at best made. Not to mention, doesn't change the fact that androids as a species are unmaintained.

**Comment by FriendlyContractor** [comment]
As much as i don't like content being removed i didnt even know two of those races existed so i think removing them is fine for what my word is worth

**Comment by Jeff-3** [comment]
I'd reconsider some of the replacements for android. The chaplain android conversion was interesting as androids had some unique interactions, but they're pretty much just fully aug'd humans with few differences. I think a more appropriate replacement would be a similar process to the robot virus, where it just fully augments your body instead of changing your race. This is a better option since it both preserves the functionality and the spirit of the mechanic.

**Comment by Fronsis** [comment]
NOOOO please not the mushroom people, i totally forgot about them, i wanted to do a gimmick and pray to get converted into one :tragedy:

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -1099,15 +1099,9 @@
 /mob/living/carbon/human/species/abductor
 	race = /datum/species/abductor
 
-/mob/living/carbon/human/species/android
-	race = /datum/species/android
```
Comment (Archanial) [inline code/modules/mob/living/carbon/human/human.dm]: > Unusued species

This one happens to be used, unless you apply some ""buffs"" to the robot species when converted in altair don't remove it.

**Review by Archanial — APPROVED** [review]
mush people my beloved

**Pull request merged**